### ISSUE TITLE
Remove numbering from elements

### ISSUE THREAD

**Issue by rlindstrm** [Xt-EHR/xt-ehr-common#5]
Common data models have different numbers in different guidelines. There is no possibility to keep a consistent and logical numbering. Let's remove the numbering. 
If we need data codes for data elements, we can have them, but they should not tempt people to order elements according to the codes.

**Comment by costateixeira** [comment]
what numbers? Those that are "A1.1" etc?

**Comment by rlindstrm** [comment]
Yes. They come from the guidelines, but they're not overly consistent already there. Maintaining any sort of logical numbering across different guidelines/use cases would not be feasible.
